operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
+### Contributing to Documentation
 
 The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
 
-Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/) - it applies to various types of contributions including documentation.
+Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
-### DocUp/Documentation Issue Process
+### Documentation MVP
+
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide. 
+
+Here are some guidelines around this:
+
+* When submitting a PR, please include updated documentation if applicable. 
+* The documentation update can be in the form of a bullet list or an outline.
+* Label the PR as **Docs Needed** and tag @justinegeffen or @amyblaise.
+* The documentation you've submitted will be reviewed.
+* At times, the editors may make and commit stylistic changes \(such as punctuation\) but any content changes will be added as a suggestion for the submitter's consideration. 
+* Once the PR is approved, it will be merged, and the documentation will be updated. 
+
+You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review). 
+
+Note: This process does not apply to the [API Documentation requirements](https://github.com/mattermost/mattermost-api-reference). 
+
+### DocUp Issues
 
 Requests for documentation can be made within [community.mattermost.com](https://community.mattermost.com), using the DocUp plugin embedded in the post menu.
 
 ![Access the Doc Up plugin by hovering over a message and clicking the &quot;...&quot; menu.](../../../.gitbook/assets/image%20%2818%29.png)
 
-When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list.
+When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list. An update is listed in the Documentation channel, with the issue link.
+
+As the issues are open to the community, the more information provided in the ticket the better. Once the link is shared in the Documentation channel, review the issue and add any links to appropriate documentation and/or Jira tickets. This ensures that the assignee is able to take on and complete the work within the turnaround times. 
```

Comment:
Should they review it in the repository instead?

Also, depending on the option you choose, the issue may be created in mattermost-handbook or mattermost-developer-documentation repos

Reply:
Good points; updated.